components/brave_wallet_ui/components/extension/confirm-transaction-panel/index.tsx

```diff
@@ -122,8 +125,30 @@ function ConfirmTransactionPanel ({
   const [isEditingAllowance, setIsEditingAllowance] = React.useState<boolean>(false)
   const [showAdvancedTransactionSettings, setShowAdvancedTransactionSettings] = React.useState<boolean>(false)
   const [maxPriorityPanel, setMaxPriorityPanel] = React.useState<MaxPriorityPanels>(MaxPriorityPanels.setSuggested)
+  const [transactionConfirmed, setTranactionConfirmed] = React.useState<boolean>(false)
+  const [confirmedTransactionNumber, setConfirmedTransactionNumber] = React.useState<number | undefined>(undefined)
```

Comment:
maybe rename to queueLength, since that is the number you are tracking here?

Reply:
Renamed!